.github/actions/setup-localstack/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Setup LocalStack
+description: Setups LocalStack to run Terraform Tests against
+inputs:
+  GITHUB_TOKEN:
+    description: Workflow GitHub Token (limited permissions)
+    required: true
```

Comment:
🔧  This isn't used in this action and can be removed.

```suggestion

```

Reply:
removed!